src/app/models/entities/concepts.ts

```diff
@@ -2,4 +2,5 @@ export interface Concept {
   code: string;
   display: string;
   include: boolean;
+  internalId?: string;
```

Comment:
added this so we can edit custom concept code/display values - otherwise we create a new concept every time we update. Shouldn't affect existing concepts elsewhere.